EasyPost/VersionInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Reflection;
+
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("2.6.0")]
+[assembly: AssemblyInformationalVersion("2.6.0")]
+
+// Version information for an assembly consists of the following four values:
```

Comment:
Let's add the missing `// [assembly: AssemblyVersion("1.0.*")]` to the end of this comment block and move the entire block comment above line 3 like it appeared in the individual files and then this will be good to go!

Reply:
Sorry about that. All set on this.